homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -89,14 +98,17 @@ def __init__(
         device: DoorDevice,
         description: AccSensorEntityDescription,
     ) -> None:
-        """Initialize a sensor for an Abode device."""
+        """Initialize a sensor for an Aladdin Connect device."""
         self._device_id = device["device_id"]
         self._number = device["door_number"]
         self._name = device["name"]
+        self._model = device["model"]
         self._acc = acc
         self.entity_description = description
         self._attr_unique_id = f"{self._device_id}-{self._number}-{description.key}"
         self._attr_has_entity_name = True
+        if self._model == "01" and description.key in ("battery_level", "ble_strength"):
```

Comment:
Should we consider making this slightly more forward compatible and use something like `self._model != "00"` (I don't know what the actual state is for the older model that doesn't suppport these sensors) so that when model "02" is released we don't need to update this?

It could be totally reasonable to just say, no we want to explicitly limit this to model 01, so this is an open suggestion.

Reply:
With my understanding I disagree with this comment.
It seems that 02 is already released, and the sensors are available on that model. So it's not possible to use the description for conditionality.

Reply:
Does model 02 not support battery_level and ble_strength sensors?   Are those a model 01 feature only?

Reply:
More specifically, is there a reason that we only want to enable these sensors by default for model 01?

Reply:
Model 01 is an add on door opener device that can control up to 4 non OHD/Genie doors.  This device has a battery operated door position sensor that uses BLE to know the status/position of the door.  IF a user has Model 01, battery level and BLE signal strength are good to know (maybe even "important").  

As far as I know, all other models are integrated door openers that do not have the ability to use the BLE device.  Unfortunately (or fortunately depending) the keys are present in the JSON message but always contain a "0" value.

Reply:
Ah okay.  Now I understand why we would want these sensors enabled only for model 01.  Consider adding a comment somewhere indicating why.  Otherwise, I'm good!